projects/burnr/package.json

```diff
@@ -14,8 +14,8 @@
     "prebuild": "yarn clean && mkdir dist && cp -r ./public/* ./dist/.",
     "build": "parcel build ./dist/index.html --public-url ./",
     "deep-clean": "yarn clean && rm -rf node_modules",
-    "clean": "rm -rf dist",
-    "dev": "yarn run clean && tsc --noEmit & parcel ./public/index.html",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "dev": "yarn run clean && tsc -b & parcel ./public/index.html",
```

Comment:
Why remove the noEmit? tsc is typechecking while parcel takes care of compiling the code. Afaiu this change will typecheck, build and then parcel will compile the code to show it for dev.

Reply:
You can't use -b and --noEmit together. I decided it would be preferable to have incremental building and detection of changes in dependencies than suppressing the build output

Reply:
Here is the justification from the typescript team on why noEmit wont build dependent references with `--noEmit` - https://github.com/microsoft/TypeScript/issues/40431

Reply:
I agree with the above. noEmit is used for not emitting compiler output files like JavaScript source code, source-maps or declarations. https://www.typescriptlang.org/tsconfig#noEmit . All I am saying is that so far we were using parcel to compile our project while keep tsc (with --noEmit) just for typechecking. By replacing --noEmit with -b we now use tsc to typecheck and compile, and then parcel to compile again.

Reply:
The point is a user cannot just clone then cd into burnr and run `yarn build` if we use `--noEmit` ... I think that is surprising.  By using `-b` instead of `--noEmit` its true that the compile is slightly slower than using `--noEmit` but its "bulletproof" .. it will always do what you expect no matter the state of the other projects (it will compile them if needed)

Reply:
Also if you make changes to `@substrate/connect` (and don't recompile it) then cd into one of the demos and then run `yarn dev` in the demo you will not have the changes in @substrate/connect since typescript with `--noEmit` won't check the dependency for changes - it will use the stale build output from `@substrate/connect`.  That's also surprising

Reply:
Regarding the slowdown by doing a full compile instead of using `--noEmit`.. that is probably only a concern for the first hit.  `-b` does incremental compilation so after the first build it will have build caches and so is able to optimise the build time depending on what has changed

Reply:
I'm not sure I follow. I just replicated the steps you mentioned (after changing the `-b` to `--noEmit` ofc) and I was able to `yarn dev` from inside burnr dir.